tests/validation/cattlevalidationtest/core/common_fixtures.py

```diff
@@ -167,3 +168,23 @@ def ping_link(src, link_name, var=None, value=None):
             time.sleep(1)
 
     assert from_link == value
+
+
+@pytest.fixture
```

Comment:
We could benefit from this function in our integration tests too! :) can you add it there once this PR is merged?